practices/managing-node-projects.md

```diff
@@ -135,3 +136,21 @@ Snyk not only provides alerts when a new vulnerability is detected in one of the
 [Using components with known vulnerabilities caused 24% of the top 50 breaches](https://snyk.io/blog/owasp-top-10-breaches/). Keeping dependencies up-to-date is therefore crucial to maintain our standards of security.
 
 Every node.js project must be added to the Springer Nature organisation in Snyk so it can be monitored for security vulnerabilities.
+
+## Handling dependencies from third parties
+
+A question that often comes up, should we commit the contents of the `node_modules` directory or any selected module. The answer on balance is that we don't recommend it. 
```

Comment:
Not sure this does "often" come up? Also there is no question mark on the end of the sentence. Suggest simply "We do not recommend checking the contents of `node_modules` into version control. It should be possible to install & deploy apps and their external dependencies reliably. Also some modules will be built differently according to the host platform."

Reply:
Happy to remove often, but without some proper introduction the section doesn't make sense.

Reply:
My alternative makes sense to me :) We have the recommendation, two good reasons why, and then some ways to implement. Previously no reason was given for the decision.